.travis.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       go: 1.12.x
       install:
         - cd webapp/cas
+        - export GO111MODULE=on
```

Comment:
That's weird that you need that, as this PR is unrelated, but I'm unable to find a better fix.